setup.py

```diff
@@ -72,6 +72,7 @@
         'knack',
         'pylint==2.11.1',
         'pytest-xdist',  # depends on pytest-forked
```

Comment:
This comment is no longer true.

https://pytest-xdist.readthedocs.io/en/latest/changelog.html#pytest-xdist-2-5-0-2021-12-10

> [#468](https://github.com/pytest-dev/pytest-xdist/issues/468): The --boxed command line argument is deprecated. Install [pytest-forked](https://pypi.org/project/pytest-forked) and use --forked instead. pytest-xdist 3.0.0 will remove the --boxed argument and pytest-forked dependency.